
float64 vx
float64 vy
float64 vz
float64 wx
float64 wy
float64 wz
---
float64 q1_dot
float64 q2_dot
float64 q3_dot

 
